defi/l2/adapters/ada.ts

```diff
@@ -1,7 +1,7 @@
-import fetch from "node-fetch";
+import { cachedFetch } from "@defillama/sdk/build/util/cache";
 
 export async function fetchAdaTokens(): Promise<any[]> {
-  const res = await fetch(`https://api.muesliswap.com/token-list`).then((r) => r.json());
+  const res = await cachedFetch({key: "muesliswap-token-list", endpoint: "https://api.muesliswap.com/token-list"})
```

Comment:
You can skip the key here if you want, it gets computed from endpoint as hash if key is missing for get requests